activesupport/lib/active_support/fork_tracker.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module ActiveSupport
+  module ForkTracker # :nodoc:
+    module CoreExt
+      def fork(*)
```

Comment:
It should be made private, otherwise it makes `fork` public for all Object instances.